timefold-solver/src/test/java/com/baeldung/timefoldsolver/ShiftScheduleConstraintProviderTest.java

```diff
@@ -0,0 +1,50 @@
+package com.baeldung.timefoldsolver;
+
+import java.time.LocalDate;
+import java.util.Set;
+
+import ai.timefold.solver.test.api.score.stream.ConstraintVerifier;
+import org.junit.jupiter.api.Test;
+
+class ShiftScheduleConstraintProviderTest {
+
+    private static final LocalDate MONDAY = LocalDate.of(2030, 4, 1);
+    private static final LocalDate TUESDAY = LocalDate.of(2030, 4, 2);
+
+    ConstraintVerifier<ShiftScheduleConstraintProvider, ShiftSchedule> constraintVerifier = ConstraintVerifier.build(
+            new ShiftScheduleConstraintProvider(), ShiftSchedule.class, Shift.class);
+
+    @Test
+    void atMostOneShiftPerDay() {
+        Employee ann = new Employee("Ann", null);
+        constraintVerifier.verifyThat(ShiftScheduleConstraintProvider::atMostOneShiftPerDay)
+                .given(
```

Comment:
The formatting/indentation looks like you are not using the formatter profile from the style guide, let me know if you need any help setting that up

Reply:
Done.